about.html

```diff
@@ -19,9 +19,7 @@
             I hope to go very far in the future developing many web pages</p>
         <p class="p3_about">Stay tuned to my page and see my progress</p>
     </div>
-    <form action="index.html"> <a id="back"><input id="botton_SeeMore" type="submit" value="Back" role="button"
-                aria-label="Back Home">
-            </input></a>
+    <a href="index.html" id="botton_SeeMore" role="button" aria-label="Back Home">Back</a>
```

Comment:
remove the ID, use classes
No need to add a role="button". This is a link that looks like a button, it's not a link that is acting as a button.